default.nix

```diff
@@ -0,0 +1,119 @@
+{ pkgs ? import ./pkgs.nix {} }:
+
+with pkgs;
+
+let
+
+  config = import ./config.nix;
+
+  name = "hp-test";
+
+  script = pkgs.writeShellScriptBin name
```

Comment:
This line creates a command in nix-shell "hp-test", but you are not using it in shellHook. I think that if you're trying to expose it it might nice to write in README.md sth like "type hp-test to run tests". Or if there is no plan to use it I think deleting this part might make sense.